README.md

```diff
@@ -427,6 +427,14 @@ Devnet can be started with the `--start-time` argument.
 starknet-devnet --start-time START_TIME_IN_SECONDS
 ```
 
+### Timeout
+
+Timeout can be passed to Devnet's HTTP server.
```

Comment:
Describe here in more detail why it could be useful.

Reply:
Also, somewhere in the readme there is a list of all CLI params, that should also be updated

Reply:
https://github.com/Shard-Labs/starknet-devnet/pull/256/commits/b715d1628e248a90cc36c49fd6aa78c9d117f1fb